languages/python/exercises/concept/comments/comments.py

```diff
@@ -0,0 +1,6 @@
+def omit_String():
```

Comment:
style nit: Python names should be lowercase, with underscores, like so: `omit_string`.

https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles